.github/workflows/main.yml

```
@@ -60,6 +60,11 @@ jobs:
     # Workaround https://github.com/nektos/act/issues/1875
     uses: apple/swift-nio/.github/workflows/static_sdk.yml@main
 
+  wasm-sdk:
+    name: WebAssembly SDK
+    # Workaround https://github.com/nektos/act/issues/1875
+    uses: apple/swift-nio/.github/workflows/wasm_sdk.yml@katei/add-wasm-ci
```

@kateinoigakukun I think to be able to test this locally with act, the following needs done.

Adding a TODO here as well just to make sure we don't forget to change the CI branch to main before merging this PR.

```suggestion
    # TODO: Switch to this line before merging:
    # uses: apple/swift-nio/.github/workflows/wasm_sdk.yml@main
    uses: kateinoigakukun/swift-nio/.github/workflows/wasm_sdk.yml@katei/add-wasm-ci
```